### ISSUE TITLE
Wigner plotting unstable

### ISSUE THREAD

**Issue by elib20** [XanaduAI/MrMustard#252]
### Before posting a bug report

- [X] I have searched exisisting GitHub issues to make sure the issue does not already exist.

### Expected behavior

Expect Wigner plots to display correctly, since it is possible in qutip.

### Actual behavior

Wigner plots do not display correctly. Likely some instability for high Fock cutoff.

### Reproduces how often

Every time for high Fock cutoff.

### System information

```shell
Mr Mustard: a differentiable bridge between phase space and Fock space.
Copyright 2021 Xanadu Quantum Technologies Inc.

Python version:            3.10.11
Platform info:             Windows-10-10.0.19045-SP0
Installation path:         N/A
Mr Mustard version:        0.5.0-dev
Numpy version:             1.23.5
Numba version:             0.56.4
Scipy version:             1.10.1
The Walrus version:        0.20.0
TensorFlow version:        2.10.0
```

### Source code

```shell
from mrmustard.utils.wigner import wigner_discretized
import numpy as np
import qutip
import matplotlib.pyplot as plt


def compare_wigner(alpha):
    state = qutip.coherent_dm(cutoff, alpha)
    W_mm, Q, P = wigner_discretized(state.full(), qvec, pvec, hbar=1)
    W_qt = qutip.wigner(state, qvec, pvec)

    plt.figure()
    plt.contourf(Q, P, W_mm, 128)
    plt.title(f"mr mustard, $\\alpha={alpha}$")

    plt.figure()
    plt.contourf(qvec, pvec, np.real(W_qt), 128)
    plt.title(f"qutip, $\\alpha={alpha}$")


cutoff = 100
qvec = np.linspace(-10, 10, 200)
pvec = np.linspace(-10, 10, 200)

compare_wigner(alpha=6)
compare_wigner(alpha=7)
```

**Comment by nquesada** [comment]
I get a similar issue. When I try to do a simple plot such as:

```python
from qutip import destroy, coherent
from mrmustard.lab import *
cutoff = 100
a = destroy(cutoff)
cutoff = 100
a = destroy(cutoff)
val = 0.3
triplet = (val*a.dag()*a.dag()*a.dag() -  val*a*a*a).expm() * coherent(cutoff, 0)
print(np.linalg.norm(triplet))
State(ket=data)
```
I get a figure that seems weird at best:

![image](https://github.com/XanaduAI/MrMustard/assets/991946/32ba6794-aa47-4fba-8810-b91fe6a582cc)

**Comment by ziofil** [comment]
Solved by @SamFerracin PR #276 now you can use:

```python
from mrmustard import settings
settings.DISCRETIZATION_METHOD = 'clenshaw'
```

and you get this 
<img width="584" alt="Screenshot 2023-10-19 at 1 42 33 PM" src="https://github.com/XanaduAI/MrMustard/assets/8944955/bee5ac8b-f633-4b4c-bc97-6010c267be08">

**Comment by nquesada** [comment]
Thanks Folks!
